src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx

```diff
@@ -61,6 +61,18 @@ export function PopulationCriteriaHome() {
     ]);
   }, [groupsBaseUrl, measure?.groups]);
 
+  const renderQdmComponents = () => {
+    if (measure) {
+      if (path.includes("/base-configuration")) {
+        return isQDM() ? <BaseConfiguration /> : history.push("/404");
+      } else if (path.includes("reporting")) {
+        return isQDM() ? <QDMReporting /> : history.push("/404");
```

Comment:
There's been discussions about moving out the model checking to reduce the amount of "if <model>" checks within a component. 

In my head, I had interpreted it to mean having model specific components over model branches within a single component.